starter-code/models/place.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const placeSchema = new Schema({
+  name: String,
+  longitude: Number,
+  latitude: Number
+});
+
```

Comment:
Indexing allows you to make faster querys by some field. In this case, you will do a lot of geographic searches, so it's a good idea to add after the schema:
```
placeSchema.index({ location: '2dsphere' });
```